database/datasource.js

```diff
@@ -60,12 +60,16 @@ const create = async (modelName, data) => {
 
 const update = async (modelName, data, options = {}) => {
   const model = getModelDefinitions()[modelName];
-  return await model.update(data, options);
+  return await model.update(data, 
+    {where: {id: options}});
```

Comment:
I think a better way to structure this would be to create a separate function named updateByPk. Then we don't lose the ability to update in bulk. Something like: 
```
const updateByPk = async (modalName, data, pk) {
  return await update(modalName, data, {
    where: {
      id: pk,
    },
  });
}
```